modules/core/src/main/scala/org/tessellation/http/routes/L0ClusterRoutes.scala

```diff
@@ -0,0 +1,24 @@
+package org.tessellation.http.routes
+
+import cats.Monad
+
+import org.tessellation.domain.cluster.services.L0Cluster
+
+import org.http4s.HttpRoutes
+import org.http4s.circe.CirceEntityCodec.circeEntityEncoder
+import org.http4s.dsl.Http4sDsl
+import org.http4s.server.Router
+
+final case class L0ClusterRoutes[F[_]: Monad](l0Cluster: L0Cluster[F]) extends Http4sDsl[F] {
+  private val prefixPath = "/l0-cluster"
+
+  private val httpRoutes: HttpRoutes[F] = HttpRoutes.of[F] {
+    case GET -> Root / "peers" =>
+      Ok(l0Cluster.getL0Peers)
```

Comment:
this should be obtained by using `cluster/info` endpoint. you can also add the response signer there if you want

Reply:
I would rather not reuse the `cluster/info` endpoint because then the adjusting to `L0Peer` would happen on the client side. Hence every cluster can be treated as `L0`. I would rather have a specific endpoint for fetching L0 peers on L0 layer.

Reply:
I think that's proper way of doing that. L1 shouldn't recognize L0 as actually L0. It should use it as a "layer". Eventually, you can create an L2 cluster that will connect to L1 layer (so no L0 peers but L1 peers). and functionality stays the same

Reply:
So to sum up - L1 should know it's being connected to L0 layer but L0 layer shouldn't expose itself as L0 directly.